arch-design/delfin/PerfomanceMontoringDesign.md

```diff
@@ -1,314 +1,313 @@
+| writeRequests     | IOPS | Write requests per second                                          |
+| readResponseTime  | ms   | Average time taken for a read  IO operation in ms                  |
+| writeResponseTime | ms   | Average time taken for a write  IO operation in ms                 |
+
+**Controller**
+
+| Metric Name       | Unit | Description                                                        |
+| ----------------- | ---- | ------------------------------------------------------------------ |
+| requests          | IOPS | Input/output operations per second                                 |
+| readThroughput    | MB/S | Represents how much data read is successfully transferred in MB/s  |
+| writeThroughput   | MB/S | Represents how much data write is successfully transferred in MB/s |
+| responseTime      | ms   | Average time taken for an IO operation in ms                       |
+| cpuUsage          | %    | Overall CPU usage in %                                             |
+| memoryUsage       | %    | Overall Memory usage in %                                          |
+| readRequests      | IOPS | Read requests per second                                           |
+| writeRequests     | IOPS | Write requests per second                                          |
+| throughput        | MB/s | Represents how much data is successfully transferred in MB/s       |
+| readResponseTime  | ms   | Average time taken for a read  IO operation in ms                  |
+| writeResponseTime | ms   | Average time taken for a write  IO operation in ms                 |
+
+**Port**
+
+| Delfin Metric     | Unit | Remarks                                                            |
+| ----------------- | ---- | ------------------------------------------------------------------ |
+| Throughput        | MB/s | Represents how much data is successfully transferred in MB/s       |
+| Requests          | IOPS | Input/output operations per second                                 |
+| responseTime      | ms   | Average time taken for an IO operation in ms                       |
+| readThroughput    | MB/s | Represents how much data read is successfully transferred in MB/s  |
+| writeThroughput   | MB/s | Represents how much data write is successfully transferred in MB/s |
+| readRequests      | IOPS | Read requests per second                                           |
+| writeRequests     | IOPS | Write requests per second                                          |
+| readResponseTime  | ms   | Average time taken for a read  IO operation in ms                  |
+| writeResponseTime | ms   | Average time taken for a write  IO operation in ms                 |
+
+**Disk**
+
+| Delfin Metric | Unit | Description                                                  |
+| ------------- | ---- | ------------------------------------------------------------ |
+| responseTime  | ms   | Average time taken for an IO operation in ms                 |
```

Comment:
Similar comment, is responseTime == latency?

Reply:
same as baove